src/api/submitOrder.js

```diff
@@ -55,7 +57,11 @@ module.exports = async (dvf, orderData) => {
     }
   }
 
-  return post(dvf.config.api + '/v1/trading/w/submitOrder', {
-    json: await dvf.createOrderPayload(value)
-  })
+  const json = await (
+    value.length
```

Comment:
no lodash :)

Reply:
sorry was wrong it is there missed it :D

Reply:
I think `ramda` is the main utility lib we miss in client and UI, I wish we added that to these projects as well to use more consistent logic across

Reply:
ramda is already a dep of [dvf-utils](https://github.com/DeversiFi/dvf-utils/blob/master/package.json#L25) we can add to the client as well

I also think that there is no point in striving to avoid adding deps like `lodash/ramda/aigle` in application code. If you are writing a small library than it might be worth making it dependency free, but in application code the benefits of consistently using libraries which elegantly solve commonly encountered problems greatly outweigh the cost of adding a few dependencies. Besides, other deps we use (like web3) are much much larger then the 3 libs combined.